trunk/SUAVE/Methods/Weights/Correlations/BWB/landing_gear.py

```diff
@@ -0,0 +1,29 @@
+# landing_gear.py
```

Comment:
There is not difference. I just copied the file over in the BWB folder for consistency in the code accessing the same directory during an analysis. Thoughts?